Classes/Package.php

```diff
@@ -18,21 +18,30 @@
 use Neos\Flow\Core\Bootstrap;
 use Neos\Flow\Package\Package as BasePackage;
 use Neos\ContentRepository\Domain\Model\Workspace;
+use Neos\Flow\Annotations as Flow;
 
 /**
  * The Neos RedirectHandler NeosAdapter Package
  */
 class Package extends BasePackage
 {
+    /**
+     * @Flow\InjectConfiguration(path="disableAutomaticRedirects")
+     * @var boolean
+     */
+    protected $disableAutomaticRedirects;
```

Comment:
I had some problems in the past when injecting configuration during bootstrap. Does it work reliably?

Reply:
We had no issues so far in dev and production contexts, but tested no edge cases so far.